src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+
+const InputTodo = ({ addTodoProps }) => {
+  const [inputText, setInputText] = useState({
+    title: '',
+  });
+
+  const onChange = (e) => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (inputText.title.trim()) {
+      // eslint-disable-next-line react/destructuring-assignment
+      addTodoProps(inputText.title);
+      setInputText({
+        title: '',
+      });
+    } else {
+      alert('Please write item');
+    }
```

Comment:
Kindly remove it in order to maintain js best practices. [Ref](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)